src/viam/sdk/components/motor/client.cpp

```diff
@@ -66,7 +66,7 @@ Motor::properties MotorClient::get_properties(const AttributeMap& extra) {
         .invoke([](auto& response) { return from_proto(response); });
 }
 
-grpc::StatusCode MotorClient::stop(const AttributeMap& extra) {
+void MotorClient::stop(const AttributeMap& extra) {
```

Comment:
I agree it can! I kept it as-is just for a separation of the PR purpose, but I'm currently working on a PR to standardize use of `client_helper` and `service_helper` everywhere, and will be updating the `stop` methods in that PR.

Reply:
Sounds good.